Celeste.Mod.mm/Content/Dialog/Polish.txt

```diff
@@ -166,6 +192,8 @@
 # Everest Updater // Aktualizator Everest
 	EVERESTUPDATER_NOTSUPPORTED=			Aktualizowanie nie jest wspierane na tej platformie - anulowanie.
 	EVERESTUPDATER_NOUPDATE=				Brak aktualizacji - anulowanie.
+	EVERESTUPDATER_CREATINGLEGACYREF=			Konfigurowanie podstawowej instalacji legacyRef
+	EVERESTUPDATER_NOTAVAILABLE=				Wymagana budowa Everest jest niedostępna!
```

Comment:
`wersja` instead of `budowa`?

Reply:
resolved in d83482e

since this is under the context of the everest updater, its likely that this refers to versions instead of individual builds. so `wersja` (version) sounds more natural here